app/src/main/res/values/setup.xml

```diff
@@ -30,6 +30,16 @@
         <item>@string/pref_value_background_6_hours</item>
     </string-array>
 
+    <string name="pref_value_theme_light" translatable="false">LIGHT</string>
```

Comment:
We still have translations for this resource, as it was translatable before. Could this lead to conflicts or will transifex remove these translations?

https://github.com/stefan-niedermann/nextcloud-deck/blob/7802386a28c58da31aeebed318315e5856f94f59/app/src/main/res/values/strings.xml#L157

Reply:
Transifex will remove them automatically, no worries 👍